cpp/include/cuspatial/experimental/ranges/multilinestring_range.cuh

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <thrust/pair.h>
+
+#include <cuspatial/cuda_utils.hpp>
+#include <cuspatial/traits.hpp>
+#include <cuspatial/vec_2d.hpp>
+
+namespace cuspatial {
+
+/**
+ * @brief Non-owning range-based interface to a multilinestring data
```

Comment:
```suggestion
 * @brief Non-owning range-based interface to multilinestring data
```

Reply:
Sorry for the one-character suggestion. (But it matches the multipoint comment this way)